src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj

```diff
@@ -61,4 +61,6 @@
       <Pack>True</Pack>
```

Comment:
The commit message doesn't match the **Impact on version** section.

Reply:
Also needed on all the others, just removed my approved tag

Reply:
So you used `chore:` but if you say that this is a bugfix, `fix:` would be more appropriate.

Reply:
Does checking Other and writing "workaround" works?

Reply:
Also changing the name of the PR has no impact.
The commit needs to be redone. I see it's still `feat:` so `gitversion` will bump the **minor** and not the **patch**.

Reply:
commit message change, updated proposed changes section